### PR TITLE
Updated the submodule and YugaByte version

### PR DESCRIPTION
**Pull request by vaibhavp**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -10,8 +10,8 @@ resource "azurerm_resource_group" "YugaByte-Group" {
 resource "azurerm_virtual_network" "YugaByte-Netowrk" {
     name                = "${var.prefix}${var.cluster_name}-VPC"
     address_space       = ["10.0.0.0/16"]
-    location            = "${var.region_name}"
-    resource_group_name = "${azurerm_resource_group.YugaByte-Group.name}"
+    location            = var.region_name
+    resource_group_name = azurerm_resource_group.YugaByte-Group.name
 
     tags = {
         environment = "${var.prefix}${var.cluster_name}"
```
Comment (ramkumarvs) [inline main.tf:17]: Typo on the below line "YugaByte-Network"

**Review by ramkumarvs — COMMENTED** [review]
Rebase the branch and update.

File: main.tf (hunk 1 above)
Comment (vaibhavp) [reply main.tf:17]: Typo removed

**Comment by bhavin192** [comment]
Closing this as we have #6 now.

**Pull request closed without merging**